ros2/diagnostics/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.8)
+project(diagnostics)
```

Comment:
The package name is quite generic. I'd recommend to have these two packages called `diagnostics_publisher_cpp` and `..._py` and have them live in the `diagnostics` folder where also the README is placed.

```
- ros2
  - diagnostics
    - diagnostics_publisher_cpp
    - diagnostics_publisher_py
    - README
```